recipe/conda_build_config.yaml

```diff
@@ -15,6 +15,9 @@ zip_keys:
     - channel_targets
     - is_rc
 
+libmagma:
+  - 2.9.0
```

Comment:
Your naming here needs to match `meta.yaml`, i.e. 
https://github.com/conda-forge/pytorch-cpu-feedstock/blob/a22d6c9d24c0d2910bdd53145c8cec352b69d92f/recipe/meta.yaml#L127

Reply:
Thanks.